tests/funcs/fs.sh

```diff
@@ -17,6 +17,16 @@ t_sync_seq_index()
 	t_quiet sync
 }
 
+#
+# Ensure async inode deletion has run and completed.
+#
+t_sync_deleted()
+{
+	t_sync_seq_index
+	t_sync_seq_index
+	sleep 10
```

Comment:
The sleep feels both too long for the expected case, and maybe not long enough depending on what previous tests might have done.  If we're using this in a test that is specifically trying to kick the work, maybe we can add counters for the work .. finishing?  And use those to see when the work has executed since the unlink and so we can assume that the other counters reflect the unlink we're testing?